platform/parser/api/src/main/java/org/codice/ddf/parser/Parser.java

```diff
@@ -41,24 +46,52 @@
      *
      * @param configurator object containing the relevant configuration information needed to
      *                     perform the conversion
-     * @param obj the root of the object graph to convert
-     * @param os the output stream on which the converted object is written
+     * @param obj          the root of the object graph to convert
+     * @param os           the output stream on which the converted object is written
      * @throws ParserException
      */
     void marshal(ParserConfigurator configurator, Object obj, OutputStream os)
             throws ParserException;
 
     /**
```

Comment:
You probably want to add a description of what this marshal does... something like `Converts an object graph into the appropriate output format, writing it to the given node.`